src/HttpRequest.cc

```diff
@@ -93,7 +93,6 @@ HttpRequest::init()
     error.clear();
     peer_login = nullptr;      // not allocated/deallocated by this class
     peer_domain = nullptr;     // not allocated/deallocated by this class
```

Comment:
These two fields are known to suffer from similar problems. We will deal with them in a dedicated followup PR.

This comment does not request any changes in this PR.